lms/templates/fields/field_textarea.underscore

```diff
@@ -8,11 +8,19 @@
     </div>
 
     <div class="u-field-value" id="u-field-value-<%- id %>" aria-labelledby="u-field-title-<%- id %>"><%
-        if (mode === 'edit') {
-            %><textarea id="u-field-textarea-<%- id %>" rows="4" aria-describedby="u-field-placeholder-value-<%- id %>"><%- value %></textarea><%
-        } else {
+        if (mode === 'edit') {%>
+            <% if (descriptionMessage) { %>
+                <textarea id="u-field-textarea-<%- id %>" rows="4" aria-describedby="u-field-description-help-<%- id %>"><%- value %></textarea>
+            <% } else { %>
+                <textarea id="u-field-textarea-<%- id %>" rows="4" aria-describedby="u-field-placeholder-value-<%- id %>"><%- value %></textarea>
+            <% } %>
+        <% } else {
```

Comment:
Actually this will be slightly more shorter:
```
var textareaDescribedBy = (message ? 'u-field-message-help-' : 'u-field-placeholder-value-') + id;
if (mode === 'edit') {
   %><textarea id="u-field-textarea-<%- id %>" rows="4" aria-describedby="<%- textareaDescribedBy %>"><%- value %></textarea><%
} else {
```